support-frontend/app/controllers/IdentityController.scala

```diff
@@ -81,14 +82,35 @@ class IdentityController(
         )
     }
   }
+
+  def createSignInURL(): Action[CreateSignInTokenRequest] = PrivateAction.async(circe.json[CreateSignInTokenRequest]) { implicit request =>
```

Comment:
Note that (by proxy) this is giving public access to an endpoint which currently requires authentication. If we ok with the endpoint that creates encrypted emails being made public, then we could remove the authentication on it and also this proxy. @guardian/identity - any thoughts?

Reply:
It's still CSRF protected, if that's any consolation

Reply:
I've given this some more thought and think that my advice in #1908 was probably overkill (apologies). 

To give you some context, we initially created a way of encrypting a user's email address for scenarios where there was no other way to forward this data than as a query parameter; for example, a link in an email. 

The other advantage of using the encrypted email token was that the sign-in page could be personalised based on the user's state. However, this is not applicable in this scenario since someone (via contributing) could obtain an encrypted email for an account that isn't theirs and then navigate to the sign-in page to infer information about the account associated with the respective email address. If this PR was merged as is, then it would be necessary to merge [#544](https://github.com/guardian/identity-frontend/pull/544) first to mitigate this issue.

Since you are already using `fetch()`, I think the simplest solution (and something I should have suggested previously) would be to include the email address in a custom header; we can do the corresponding work on identity-frontend to utilise this header to autofill the email field. This solution is in accordance with the [advice in the OWASP docs](https://owasp-aasvs.readthedocs.io/en/latest/requirement-9.3.html).

Assuming other people agree, I can catch up with you later to pair on this.

Reply:
@Mullefa - Good thinking around inferring sign in details, but what level of risk do you think this has? From our perspective, it is pretty important to be sending users to profile.theguardian.com using the email address they've just used to make a contribution so our copy will definitely be accurate for them.

In short: I'm wondering if you think this is a 'blocking' level of risk or if you think we can move forward with this and switch over to sending a header with their email when your side is ready to process that header.

Reply:
Isn't the work needed in `identity-frontend` to support an email header to prefill the email mostly figuring out where add some tests + https://github.com/guardian/identity-frontend/compare/lm-email-param  ? I'm still not super familiar with identity-frontend so that could definitely be wrong.

Reply:
@Mullefa @lmath I don't think you can actually route to a new page using fetch and I don't think one can send headers with a link - I could send a get request to profile.theguardian.com/signin with the header set (using fetch) but it would retrieve the document - not route the user there

Reply:
"The other advantage of using the encrypted email token was that the sign-in page could be personalised based on the user's state."

@Mullefa out of interest, what use case is this? The sign in page is unauthenticated, so I'm curious as to what user state info you would consider exposing to someone who isn't logged in (as in I'm genuinely curious, not saying I don't think there is anything that you could show),.

Reply:
Hey Jonny - yeah, good question. These have previously been used as query parameters in email links. So a user would only be able to see the customised page if they owned the respective email account.

Reply:
Given the points @jlieb10 made regarding headers and having spoken to @jlieb10 and @jfsoul, the current plan is to roll out implemented as is. As prerequisites to the merge, I'm going to update identity API so that the secret used to encrypt a user's email is not used to encrypt any other data and merge [#544](https://github.com/guardian/identity-frontend/pull/544).

Reply:
Marking this as resolved- thanks- please don't forget to review/approve @Mullefa @lmath